### ISSUE TITLE
Docs permissions: Non-members of group able to edit

### ISSUE THREAD

**Issue by bree-z** [cuny-academic-commons/openlab-theme#239]
If I'm logged into the site, I'm able to edit docs in groups I'm not a member of.

E.g., while logged into test student, I was able to edit this doc and save my changes:

http://openlabdev.commonsinabox.org/groups/how-to-properly-spit-cherry-pits/docs/steps/

![nonmember_profile](https://user-images.githubusercontent.com/7189824/42759135-900349f6-890e-11e8-9a72-1af99ea3572c.png)

<img width="1218" alt="docs_editlink" src="https://user-images.githubusercontent.com/7189824/42759143-97606ff8-890e-11e8-9cd7-d28639e1c752.png">

![nonmember_editingdoc](https://user-images.githubusercontent.com/7189824/42759152-9d39d342-890e-11e8-8096-60f692475b6c.png)

**Comment by boonebgorges** [comment]
Non-members are allowed to edit because the Settings are configured in such a way as to allow any logged-in user to edit. So this in itself is not a bug.

The bug (or unexpected behavior, I'd say) is that the settings are configured this way by default. The fact that the default settings work like this due to a complex combination of customizations on openlab-theme and the CBOX-OL package. I've just pushed up some changes that ought to address this. Please try creating a new doc, and ensure that it gets the settings you'd expect. For Public groups - the group type in question here, I think - the proper setting for "edit" should be "Members of <group name>".

**Comment by bree-z** [comment]
Thanks, Boone.  These settings make sense, and when I create a new doc, these are indeed the settings for the new doc.  However, they don't seem to be applied properly to the docs.  I'm still able to edit the doc as a non-member, even though the settings say only members can edit.  

I tried in the group I was testing with above, and also a new group I created yesterday, after you made this change.  I had the same result in both.

http://openlabdev.commonsinabox.org/groups/how-to-properly-spit-cherry-pits/docs/new-doc-new-settings/
http://openlabdev.commonsinabox.org/groups/quick-course/docs/testing-settings/

In each case, I created the doc as Test Faculty, the only member of the group.  But, I was able able to edit as Test Student, a non-member.

![docssettings](https://user-images.githubusercontent.com/7189824/43259870-73172090-90e0-11e8-8fd8-ad75638fcb08.png)

![docsteststudent png](https://user-images.githubusercontent.com/7189824/43259879-77a71d22-90e0-11e8-84eb-8969aa3f88dc.png)
